storage/innobase/xtrabackup/test/t/bug1658692.sh

```diff
@@ -0,0 +1,66 @@
+########################################################################
+# Bug #1658692: Missing .isl file for general tablespace from backup
+########################################################################
+
+. inc/common.sh
+
+require_server_version_higher_than 5.7.0
+start_server
+
+target_dir="$topdir/backup-full"
+incremental_dir="$topdir/backup-inc"
+datafile="$topdir/out_datadir.ibd"
+
+run_cmd $MYSQL $MYSQL_ARGS test <<EOF
+create tablespace out_datadir add datafile '$datafile';
+EOF
+
+count=`ls $mysql_datadir/*.isl | wc -l`
+vlog  "$count .isl files in datadir, expecting 1"
+test $count -eq 1
+
+vlog "Performing a backup to $target_dir"
+xtrabackup \
+    --backup \
+    --target-dir=$target_dir \
+    --host=127.0.0.1 \
```

Comment:
we don't need to specify host and port. xtrabackup is a wrapper function defined `common.sh`, it passing all needed connection parameters